### DISCUSSION TITLE
recipes/sauces-and-dressings/new-york-times%27-classic-marinara-sauce/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1175] [Comments]
# recipes/sauces-and-dressings/new-york-times%27-classic-marinara-sauce/

A collection of my recipes.

https://nicholaswilde.io/recipes/sauces-and-dressings/new-york-times%27-classic-marinara-sauce/

**Comment by DaveBKC** [comment]
Step two says to heat the oil. What oil and how much?

**Reply by nicholaswilde** [reply]
1/4 cup extra-virgin olive oil. I will update page.

**Reply by nicholaswilde** [reply]
It should now be fixed. I've also updated other things in the recipe.
